website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1837,14 +1837,15 @@ var Range = ace.require('ace/range').Range;
     };
 
     commandProto.addLinkDef = function (chunk, linkDef) {
-
+        

```

Comment:
The whitespace changes are a little distracting. Good practice is to configure one's text editor to trim trailing whitespace for most file types.